tests/resources/test_predictor.py

```diff
@@ -70,127 +65,6 @@ def test_delete():
         pc.delete(uuid.uuid4())
 
 
-def test_archive_and_restore(valid_label_fractions_predictor_data):
-    session = FakeSession()
-    pc = PredictorCollection(uuid.uuid4(), session)
-    entity = deepcopy(valid_label_fractions_predictor_data)
-    entity["metadata"]["archived"] = entity["metadata"]["created"]
-    session.set_responses(None, entity, paging_response(entity))
-
-    with pytest.deprecated_call():
-        archived_predictor = pc.archive(uuid.uuid4())
-    assert archived_predictor.is_archived
-
-    del entity["metadata"]["archived"]
-    session.set_responses(None, entity, paging_response())
-    
-    with pytest.deprecated_call():
-        restored_predictor = pc.restore(uuid.uuid4())
-
-    assert not restored_predictor.is_archived
```

Comment:
I'm confused as to how archiving works now and how it should be tested. There are some lines in the `update` method of `resources/predictor.py` that are no longer covered by tests and pertain to the `._archived` property.

Reply:
The `_archived` property was introduced for backwards compatibility in archiving. Instead of setting `archived` on the predictor and updating it, users can now call the `archive`/`restore` methods. Since setting `archived` emitted a deprecation warning, we can now remove it (as you already did), along with the `_archived` property, and no longer do any archiving in the `update` method.